js/src/lib/interfaces/Libraries.ts

```diff
@@ -239,6 +239,14 @@ nlp = spacy.load("${nameWithoutNamespace(model.id)}")
 import ${nameWithoutNamespace(model.id)}
 nlp = ${nameWithoutNamespace(model.id)}.load()`;
 
+const stanza = (model: ModelData) =>
+	`from stanza import Pipeline
+
+stanza.download("${nameWithoutNamespace(model.id)}")
+nlp = stanza.Pipeline("${nameWithoutNamespace(model.id)}") 
```

Comment:
In stanza you just need the language, so you can also remove `stanza-` from the id. 

```
import stanza
stanza.download('en')
nlp = stanza.Pipeline('en')

Reply:
see below :)